.travis.yml

```
@@ -16,7 +16,7 @@ before_install:
 jobs:
   include:
     - stage: Build docker image
-      script: ./scripts/buildDockerImage.sh -n $ECR_REPOSITORY_NAME
+      script: ./scripts/buildDockerImage.sh -n $IMAGE_NAME -r $AWS_DEFAULT_REGION
     - stage: Execute Gatling test
       script: ./scripts/runLoadtestOnAWS.sh -r ${S3_BUCKET} -c 2 -u 10 -d 1 -ramp 10 -cl $ECS_CLUSTER
```

Now needs extra arguments:
```suggestion
       script: ./scripts/runLoadtestOnAWS.sh -r ${S3_BUCKET} -c 2 -u 10 -d 1 -ramp 10 -cl $ECS_CLUSTER -rgn $AWS_DEFAULT_REGION -n $IMAGE_NAME
```

---

Fixed.